components/KiboOrderLineItems.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div>
+    <div class="order-shipment">
+      <div v-if="shipItems && shipItems.length > 0">
+        <div class="order-shipment__title">
+          <slot name="ship-title">
+            <h3 class="sf-heading__title h3">
+              {{ $t("Shiping to home") }}
+            </h3>
+          </slot>
+        </div>
+        <slot name="ship-sub-section" />
+
+        <div v-for="(item, index) in shipItems" :key="index" class="order-shipment__item">
+          <KiboOrderLineItem :item="item" />
+        </div>
+      </div>
+
+      <div v-if="pickupItems && pickupItems.length > 0">
+        <div class="order-shipment__spacer"></div>
+
+        <div class="order-shipment__title">
+          <slot name="pickup-title">
+            <h3 class="sf-heading__title h3">
+              {{ $t("Pickup In Store") }}
+            </h3>
+          </slot>
+        </div>
+        <slot name="pickup-sub-section" />
+
+        <div v-for="(item, index) in pickupItems" :key="index" class="item">
+          <KiboOrderLineItem :item="item" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { PropType } from "@vue/composition-api"
+import { checkoutGetters } from "@/lib/getters"
+import { Order } from "@/server/types/GraphQL"
+
+export default {
+  name: "OrderLineItems",
+  components: {},
+  props: {
+    order: {
+      type: Object as PropType<Order>,
+      default: () => ({}),
+    },
+  },
+  setup(props) {
+    const pickupItems = computed(() => {
+      return checkoutGetters.getPickupItems(props.order)
+    })
+    const shipItems = computed(() => {
+      return checkoutGetters.getShipItems(props.order)
+    })
```

Comment:
Can we make it simple one liner?
ex. const shipItems = computed(() => checkoutGetters.getShipItems(props.order) )

Reply:
Done